core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -188,7 +188,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
       }
     }
 
-    def abortActiveWorkflows(submissionId: UUID) = {
+    def abortActiveWorkflows(submissionId: UUID): Future[Seq[Future[(Option[String], Try[ExecutionServiceStatus])]]] = {
```

Comment:
I know you didn't change this function, but that type signature is a bit of a red flag to me. Furthermore it doesn't look like we're mapping over the inner `Seq[Future[...]]` (just the other Future) so it seems like `executionServiceCluster.abort` could be executing concurrently with `queryForWorkflowStatuses`, `executionServiceCluster.outputs`. Will think about how to clean that up...